recipes/conjure-client/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and paster from another recipe, but not really needed.
+{% set name = "conjure-client" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2c806c0849d24e16d0e07014c4fe4b17f4fb3480d5f2f34ab329c87ccf5d372d
+
+build:
+  noarch_python: True
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  run:
```

Comment:
you need a 

```host:
    - python
    - pip
```

section here.  Otherwise the pip that runs is the wrong one and nothing ends up getting packaged.